pkgs/development/libraries/p11-kit/default.nix

```diff
@@ -63,6 +63,13 @@ stdenv.mkDerivation rec {
     ]))
   ];
 
+  preCheck = ''
```

Comment:
Also two nits:
- Please always use the full commit id (`3ca33e5ef4f8c3e24013905cf741330c9ace3a6c`) so it does not get ambiguous as Nixpkgs grows.
- Please move `preCheck` after `postPatch` (Freedesktop and GNOME teams are trying to follow https://github.com/jtojnar/nixpkgs-hammering/blob/main/explanations/attribute-ordering.md).

Reply:
No problem, thanks for the suggestions! 🙂